mackerel-plugin-docker/lib/docker.go

```diff
@@ -37,7 +37,7 @@ var graphdef = map[string]mp.Graphs{
 	},
 	"docker.memory.#": {
 		Label: "Docker Memory",
-		Unit:  "integer",
+		Unit:  "bytes",
```

Comment:
`docker.memory.#.{cache,rss}` are taken from cgroup `memory.stat`

https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt
```
memory.stat file includes following statistics

# per-memory cgroup local status
cache		- # of bytes of page cache memory.
rss		- # of bytes of anonymous and swap cache memory (includes
		transparent hugepages).
```

Reply:
btw, in many Linux distributions these stats are always zero (but in Docker for Mac non-zero).
ref: https://docs.docker.com/engine/admin/runmetrics/#memory-metrics-memorystat
```
Therefore, many distros chose to not enable it by default.
```